pkg/cli/debug.go

```diff
@@ -753,6 +754,11 @@ $ curl -X POST  'http://localhost:8080/ts/query' \
   -H 'Accept: application/json' \
   -H 'Content-Type: application/x-protobuf' \
   --data-binary @<file>
+
+If --dump-descriptor-set is specified, the tool will output a FileDescriptorSet
```

Comment:
I think this should be its own debug command instead of hiding as a sub-option on `decode-proto`